app/src/main/java/com/kickstarter/models/CommentThread.java

```diff
@@ -0,0 +1,57 @@
+package com.kickstarter.models;
+
+import android.os.Parcelable;
+import androidx.annotation.Nullable;
+import com.kickstarter.libs.qualifiers.AutoGson;
+import org.joda.time.DateTime;
+import java.util.Objects;
+import auto.parcel.AutoParcel;
+
+@AutoGson
+@AutoParcel
+public abstract class CommentThread implements Parcelable{
+  public abstract User author();
+  public abstract String body();
+  public abstract DateTime createdAt();
+  public abstract Boolean deleted();
+  public abstract long id();
+  public abstract String cursor();
+  public abstract long parentId();
+
+  @AutoParcel.Builder
+  public abstract static class Builder {
+    public abstract Builder author(User __);
+    public abstract Builder body(String __);
+    public abstract Builder createdAt(DateTime __);
+    public abstract Builder deleted(Boolean __);
+    public abstract Builder id(long __);
+    public abstract Builder cursor(String __);
+    public abstract Builder parentId(long __);
+    public abstract CommentThread build();
+  }
+
+  public static CommentThread.Builder builder() {
+    return new AutoParcel_CommentThread.Builder();
+  }
+
+  public abstract CommentThread.Builder toBuilder();
+
+  @Override
+  public boolean equals(final @Nullable Object obj) {
+    boolean equals = super.equals(obj);
+
+    if (obj instanceof CommentThread) {
+      final CommentThread other = (CommentThread) obj;
+      equals = Objects.equals(this.id(), other.id()) &&
+              Objects.equals(this.author(), other.author()) &&
+              Objects.equals(this.parentId(), other.parentId());
```

Comment:
you need to add all the fields for the Model the next ones are missing
String body();
 DateTime createdAt();
Boolean deleted();
String cursor();

Reply:
ok.